index.html

```diff
@@ -860,6 +816,7 @@ <h3>Hyderabad International Convention Centre</h3>
   <script src="js/canvas.js"></script>
   <script src="js/content.js"></script>
   <script src="js/timeline.js"></script>
+  <script src="js/sponsor.js"></script>
```

Comment:
You can comment this for now since we are not ready with the sponsors list yet. This feature is working and ready to be merged.

Reply:
:'(...Okay I'll close the Pr